adapters/Nend/NendAdapter/nativeads/GADMAdapterNendNativeAd.m

```diff
@@ -0,0 +1,137 @@
+//
+//  GADMAdapterNendNativeAd.m
+//  NendAdapter
+//
+//  Copyright © 2019 FAN Communications. All rights reserved.
+//
+
+#import "GADMAdapterNendNativeAd.h"
+#import "GADMAdapterNend.h"
+#import "GADMAdapterNendConstants.h"
+#import "GADMAdapterNendNativeAdLoader.h"
+
+@interface GADMAdapterNendNativeAd () <NADNativeDelegate>
+
+@property(nonatomic, strong) NADNative *nativeAd;
+@property(nonatomic, strong) GADNativeAdImage *mappedIcon;
+@property(nonatomic, strong) NSArray *mappedImages;
+@property(nonatomic, strong) UILabel *advertisingExplicitlyView;
+@property(nonatomic, strong) UIImageView *imageView;
+
+@end
+
+@implementation GADMAdapterNendNativeAd
```

Comment:
This class is missing `mediaContentAspectRatio`

https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediatedUnifiedNativeAd.html#mediacontentaspectratio

Reply:
I'll add `mediaContentAspectRatio` getter.